src/main/java/org/apache/cassandra/sidecar/MainModule.java

```diff
@@ -304,4 +314,28 @@ public ChecksumVerifier checksumVerifier(Vertx vertx)
     {
         return new MD5ChecksumVerifier(vertx.fileSystem());
     }
+
+    @Provides
+    @Singleton
+    @Named("SidecarVersion")
+    private static String sidecarVersion()
```

Comment:
nit: drop `static`. I do not think it is required. The scope is singleton already.

Does the annotation work with `private` methods? It feel counterintuitive semantically, even if it works. Can you change it to `public`?

Reply:
Missed that when copy-pasting, fixed.